src/containers/RandomCocktails.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import axios from "axios";
+
+class RandomCocktail extends React.Component {
```

Comment:
I think you can change it with functional component

Reply:
ok done